api/views.py

```diff
@@ -145,13 +147,17 @@ def _formatted_bad_request(format_):
 @api_view(['GET'])
```

Comment:
please use the constant GET from consts.py

Reply:
done. but i can't understand the intuition behind storing it as a const in other file. i mean the variable name and its value is equal na

Reply:
strings constant are error prone. If you make a little mistake, you won't find it until it's too late. It is cleaner to avoid to write the same string more than once if possible.

Putting them in a clean file, where they can also be shared and imported from, is just sintactic sugar and, even if with a litlle project like this the benefit is almost 0, in the long term you can save possible mistakes + you have cleaner and better readable code.

Reply:
Wow never thought this way. Really learning a lot by contributing to these projects.